docs/content/configuration/configuring_hostdefiner.md

```diff
@@ -0,0 +1,12 @@
+# Configuring the host definer
+
+Some of the parameters within the HostDefiner custom resource are configurable. Use this information to help decide whether the parameters for your storage system need to be updated.
+
+For more information about using the host definer, see [Using dynamic host definition](../using/using_hostdefinition.md).
+    
+|Field|Description|
+|---------|--------|
+|`prefix`|Adds a prefix to the hosts defined by the host definer. The maximum prefix length is 20 characters.|
+|`connectivityType`|Selects the connectivity type for the host ports.<br>Possible input values are:<br>- `nvmeofc` for use with NVME over Fibre Channel connectivity<br>- `fc` for use with Fibre Channel over SCSI connectivity<br>- `iscsi` for use with iSCSI connectivity<br>By default, this field is blank and the host definer selects the first of available connectivity types on the node, according to the following hierarchy: NVMe, FC, iSCSI.|
```

Comment:
shouldn't it be NVMe?

Reply:
I was so sure that this section had been updated.
I fixed it. please check again here:
https://github.com/IBM/ibm-block-csi-driver/blob/docs_release-1.10.0/docs/content/configuration/configuring_hostdefiner.md
thanks

Reply:
no.
its 'nvmeofc' as I said [here](https://github.com/IBM/ibm-block-csi-driver/pull/532#discussion_r923244982)

Reply:
fixed (again)

Reply:
tnx